doc/admin/deployment.md

```diff
@@ -43,14 +81,14 @@ while still utilizing the default locations is to create the
 required directories manually. To do this execute the following commands.
 
 daos_server:
-```
+```bash
 $ mkdir /var/run/daos_server
 $ chmod 0755 /var/run/daos_server
 $ chown user:user /var/run/daos_server (where user is the user you
     will run daos_server as)
 ```
```

Comment:
I have found that I need to also mkdir /tmp/daos_sockets, but I don't know if that's just me, or if that is a common step that everyone needs to do as well.

Reply:
It depends on whether you have changed the default path for drpc/unix socket.